src/components/ImagePreviewDialog/ImagePreviewDialog.tsx

```diff
@@ -0,0 +1,26 @@
+import React, { memo, VFC } from 'react';
+import { Modal, ModalContent, ModalOverlay, Image } from '@chakra-ui/react';
+
+interface Props {
+
+  /** Image url. */
+  readonly image: string;
+
+  /** Close modal handler. */
+  readonly onClose: () => void;
+
+  /** Flag that opens/closes dialog. */
+  readonly isOpen: boolean;
+}
+
+/** Dialog for previewing images. */
+const ImagePreviewDialogComponent: VFC<Props> = ({ image, onClose, isOpen }) => (
+  <Modal isOpen={isOpen} onClose={onClose} size="2xl" isCentered>
+    <ModalOverlay />
+    <ModalContent>
+      <Image src={image} />
+    </ModalContent>
+  </Modal>
```

Comment:
I think it should be image carousel, instead of single image

Reply:
Maybe implement it somehow like [here](https://codesandbox.io/s/dd8vn?file=/src/ChakraCarousel.js)

Reply:
Nevermind, let's implement in another [task](https://github.com/vndbre/vndbre/issues/67)